comment/urls.py

```diff
@@ -18,6 +19,7 @@
     path('<int:pk>/flag/', SetFlag.as_view(), name='flag'),
     path('<int:pk>/flag/state/change/', ChangeFlagState.as_view(), name='flag-change-state'),
     re_path(r'^confirm/(?P<key>[^/]+)/$', ConfirmComment.as_view(), name='confirm-comment'),
+    path('toggle-follow/', ToggleFollowView.as_view(), name='toggle-follow'),
```

Comment:
`toggle-subscription` instead of `toggle-follow` seems more grammatically correct to me.